ui/packages/catalog-ui-search/src/main/webapp/react-component/map-info/formatting.tsx

```diff
@@ -22,9 +22,16 @@ const usngPrecision = 6
 
 import { Attribute, Coordinates, Format, validCoordinates } from '.'
 
-export const formatAttribute = ({ name, value }: Attribute): string => {
-  const isDate = metacardDefinitions.metacardTypes[name].type === 'DATE'
-  return `${name.toUpperCase()}: ${
+export const formatAttribute = ({ name, value }: Attribute): string | null => {
+  const definition = metacardDefinitions.metacardTypes[name]
+  if (definition === undefined) {
```

Comment:
:pencil2: Might want to keep this logic in container. As is, if the definition is undefined there will still be a div rendered for it here: https://github.com/codice/ddf/blob/72eabb7c4571777273eb652ce1b8439d40598b0e/ui/packages/catalog-ui-search/src/main/webapp/react-component/map-info/presentation.tsx#L53